jetbrains-core/tst/software/aws/toolkits/jetbrains/services/ecs/exec/RunCommandDialogTest.kt

```diff
@@ -30,19 +48,37 @@ class RunCommandDialogTest {
     @JvmField
     val tempFolder = TemporaryFolder()
 
+    @Rule
+    @JvmField
+    val credentialManager = MockCredentialManagerRule()
+
     private val clusterArn = "arn:aws:ecs:us-east-1:123456789012:cluster/cluster-name"
     private val serviceArn = "arn:aws:ecs:us-east-1:123456789012:service/service-name"
     private val taskArn = "arn:aws:ecs:us-east-1:123456789012:task/task-name"
     private val ecsService = Service.builder().serviceArn(serviceArn).clusterArn(clusterArn).build()
 
     private val containerDefinition = ContainerDefinition.builder().name("sample-container").build()
     private val container = ContainerDetails(ecsService, containerDefinition)
-
+    lateinit var connectionSettings: ConnectionSettings
     val command = "ls"
     private val task = Task.builder().clusterArn(clusterArn).taskArn(taskArn).build()
     private val taskList = listOf(task.taskArn())
     private val containerName = containerDefinition.name()
     private val verifyCommand = "ecs execute-command --cluster $clusterArn --task $taskArn --command $command --interactive --container $containerName"
+    private val dummyRegion = anAwsRegion()
+
+    @Before
+    fun setup() {
+        val accountSettings = MockAwsConnectionManager.getInstance(projectRule.project)
+        val dummyCredential = aCredentialsIdentifier(defaultRegionId = dummyRegion.id)
+        val credentials = credentialManager.addCredentials(
```

Comment:
The default args for addCredentials will make you all of this and return the CredentialIdentifier